.github/pull_request_template.md

```diff
@@ -1,15 +1,16 @@
-# Description:
-<!-- Summary of change, including the issue to be addressed. -->
+ Description
+ ===========
 
-## Type of change:
-<!-- Put an `x` in the box that applies. -->
-- [ ] Bug fix.
-- [ ] New feature.
-- [ ] Documentation update.
 
-## Checklist for the reviewer:
-<!-- Put an `x` in the boxes that apply. These can be filled by reviewer after the PR is created. -->
+Type of change
```

Comment:
Personally, I prefer using #. But I guess this stylistic choice is agreed upon in the team